native/acter/src/api/verification.rs

```diff
@@ -23,46 +20,37 @@ use matrix_sdk_base::ruma::{
         room::message::{MessageType, OriginalSyncRoomMessageEvent},
         AnyToDeviceEvent, EventContent,
     },
-    OwnedDeviceId, OwnedUserId,
+    OwnedDeviceId, OwnedUserId, UserId,
 };
 use std::{
     collections::HashMap,
     ops::Deref,
     sync::Arc,
     time::{Duration, SystemTime, UNIX_EPOCH},
 };
-use tokio::sync::Mutex;
+use tokio::sync::broadcast::{channel, Receiver, Sender};
+use tokio_stream::wrappers::BroadcastStream;
 use tracing::{error, info};
 
 use super::{client::Client, common::DeviceRecord, RUNTIME};
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 pub struct VerificationEvent {
-    client: SdkClient,
-    controller: VerificationController,
     event_type: String,
     flow_id: String,
-    sender: OwnedUserId,
+    sender: String,
     /// event content
     content: HashMap<String, String>,
     /// emoji array
     emojis: Vec<VerificationEmoji>,
 }
```

Comment:
`impl Stream<Item = VerificationEvent>` needs for `VerificationEvent` to derive `Default` trait.
I removed `client` and `controller` fields from `VerificationEvent`, because they didn't derive `Default`.

Reply:
this seems wrong. why does it need to be `Default`?

Reply:
It needs `Default` trait.
![image](https://github.com/user-attachments/assets/7c53f062-4abc-497f-9bdd-affcc1169269)

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2411/commits/2e2689406697bbf8c3a46f50fe2bafc7a7d4789b